src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlEnclaveAttestationParameters.cs

```diff
@@ -54,7 +54,7 @@ private byte[] Clone(byte[] arrayToClone)
         }
 
         /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlEnclaveAttestationParameters.xml' path='docs/members[@name="SqlEnclaveAttestationParameters"]/ctor/*' />
-        internal SqlEnclaveAttestationParameters(int protocol, byte[] input, ECDiffieHellmanCng clientDiffieHellmanKey)
+        internal SqlEnclaveAttestationParameters(int protocol, byte[] input, ECDiffieHellman clientDiffieHellmanKey)
```

Comment:
Looks like `SqlEnclaveAttestationParameters` can be merged now too.

Reply:
Done. It had to be split into Crypto and a NotSupported version for netstandard2.0, both are in the shared location now. I also did some cleanup so please review.

Reply:
`netfx/src/Microsoft/Data/SqlClient/SqlEnclaveAttestationParameters.cs` can be removed.

Reply:
Done.